Find a functionally equivalent code:android.provider.Settings.Global.putStringForUser:COMMENT
Method Modifier: public      static      hidden      
Comment:/**
 * @hide
 */

@@ -2,6 +2,11 @@
     if (LOCAL_LOGV) {
         Log.v(TAG, "Global.putString(name=" + name + ", value=" + value + " for " + userHandle);
     }
+    // Global and Secure have the same access policy so we can forward writes
+    if (MOVED_TO_SECURE.contains(name)) {
+        Log.w(TAG, "Setting " + name + " has moved from android.provider.Settings.Global" + " to android.provider.Settings.Secure, value is unchanged.");
+        return Secure.putStringForUser(resolver, name, value, userHandle);
+    }
     return sNameValueCache.putStringForUser(resolver, name, value, userHandle);
 }

